src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```
@@ -154,6 +154,14 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <CreateDirectory Include="$(LocalInstallationOutputPath)" />
   </ItemGroup>
 
+  <!-- Workaround until https://github.com/dotnet/core-eng/issues/7970 is resolved -->
+  <PropertyGroup>
+    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar.exe</TarCommand>
+    <TarCommand Condition="'$(OS)' == 'Windows_NT' and !Exists($(TarCommand))">'C:\Program Files\Git\usr\bin\tar.exe'</TarCommand>
+    <TarCommand Condition="'$(OS)' == 'Windows_NT' and !Exists($(TarCommand))">'C:\Program Files (x86)\Git\usr\bin\tar.exe'</TarCommand>
+    <TarCommand Condition="'$(OS)' != 'Windows_NT'">tar</TarCommand>
```

```suggestion
    <TarCommand>tar</TarCommand>
    <TarCommand Condition="'$(OS)' == 'Windows_NT'">C:\Windows\System32\tar.exe</TarCommand>
    <TarCommand Condition="'$(OS)' == 'Windows_NT' and !Exists('$(TarCommand)')">C:\Program Files\Git\usr\bin\tar.exe</TarCommand>
    <TarCommand Condition="'$(OS)' == 'Windows_NT' and !Exists('$(TarCommand)')">C:\Program Files (x86)\Git\usr\bin\tar.exe</TarCommand>
```

Single quotes aren't going to work w/ `cmd` but are useful when calling `Exists(...)`. And, this simplifies things a bit.